### PR TITLE
Modified 7 scripts.

### PR DESCRIPTION
**Pull request by MintooXavier**
Modified 7 scripts. Please review.

### CODE REVIEW

**Comment by mmcev106** [comment]
@MintooXavier, I'm seeing the following failure on B.4.9.0100:
image

Is that test succeeding dependent on an rctf PR?

**Comment by mmcev106** [comment]
@4bbakers, this PR affects manual testing for the following features.  They all pass via cypress, and the changes look like positive improvements on the manual testing side as well.  Would you like take a look and make sure you agree before I merge this?

- Feature Tests/B/Data Access Groups_10/B.2.10.0400. - User and Record Restrictions.feature
- Feature Tests/B/Data Import_16/B.3.16.1200. - Overwrite Existing Data.feature
- Feature Tests/B/Direct Data Entry - Data Collection Instrument_14/B.3.14.0100. - Create Record.feature
- Feature Tests/B/Direct Data Entry - Data Collection Instrument_14/B.3.14.1000. - Delete Form Data.feature

File: Feature Tests/B/Direct Data Entry - Data Collection Instrument_14/B.3.14.1000. - Delete Form Data.feature
Comment (4bbakers) [inline Feature Tests/B/Direct Data Entry - Data Collection Instrument_14/B.3.14.1000. - Delete Form Data.feature:1]: This one needs to keep the ##VERIFY_DE steps (line 58 to 66).  I tis verifying that the deleted data is gone.

**Review by 4bbakers — CHANGES_REQUESTED** [review]
B.3.14.1000. needs to keep the ##VERIFY_DE portion (line 58 to 66)

File: Feature Tests/B/Direct Data Entry - Data Collection Instrument_14/B.3.14.1000. - Delete Form Data.feature
Comment (4bbakers) [reply Feature Tests/B/Direct Data Entry - Data Collection Instrument_14/B.3.14.1000. - Delete Form Data.feature:1]: Disregard.  I see that it was moved into a different order.

**Review by 4bbakers — APPROVED** [review]

**Comment by mmcev106** [comment]
@4bbakers & I discussed this via slack.  We now both agree that the rewritten version of `#VERIFY_DE` a few lines up is an improvement all around.  @4bbakers said she approved this PR in our slack conversation.

**Review by mmcev106 — APPROVED** [review]
@MintooXavier, these changes are all a step in the right direction, so I'm going to go ahead and approve & merge this.  Perhaps we can move the discussion for B.4.9.0100 into that issue or a new PR.  Thank you!

**Pull request merged**